docker/kubernetes/key-registry.yaml

```diff
@@ -0,0 +1,13 @@
+---
+alice.pub:
+  public_key: alice.pub
+  associated_node_id: acme
+  metadata:
+    gameroom/first-name: alice
+    gameroom/organization: Acme
```

Comment:
For consistency sake, should `key-registry` organization value be the same as `node-registry` value for Acme?